CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/) and this project adheres to [Semantic Versioning](https://semver.org/).
 
+## [2.1.0](https://github.com/philips-software/amp-devcontainer/compare/amp-devcontainer-v2.0.0...amp-devcontainer-v2.1.0) (2023-01-31)
+
+
+### Features
+
+* Build multi-platform image ([#4](https://github.com/philips-software/amp-devcontainer/issues/4)) ([839fb0d](https://github.com/philips-software/amp-devcontainer/commit/839fb0dc61051b25926ac847bfe12646284a31a7))
+
+
+### Bug Fixes
+
+* Set executable bit on shell scripts ([33406a3](https://github.com/philips-software/amp-devcontainer/commit/33406a3d936cb0da465e9df0f27415f4a861d59d))
+* Update GITHUB_TOKEN permissions ([a4a2e1a](https://github.com/philips-software/amp-devcontainer/commit/a4a2e1a4921292beed9810fa9c099e56069cbdcb))
+* Update GITHUB_TOKEN permissions ([90ca544](https://github.com/philips-software/amp-devcontainer/commit/90ca54409b0c8b20c334e9a2a284647103f76af1))
+
 ## [Unreleased]
```

Comment:
Please remove the unreleased section

Reply:
The release-please action does a force push, so I have created PR #6 to fix the issues.